src/status_im/ui/screens/wallet/transactions/subs.cljs

```diff
@@ -146,16 +147,14 @@
         100
         (* 100 (/ confirmations max-confirmations))))))
 
-(reg-sub
-  :contacts-by-address
+(reg-sub :contacts-by-address
```

Comment:
it's better to change `reg-sub` to `re-frame/reg-sub`, but it's definitely out of the scope of this task.